akka-stream/src/main/scala/akka/stream/scaladsl/Hub.scala

```diff
@@ -1073,11 +1078,9 @@ private[akka] class PartitionHub[T](
         case RegistrationPending ⇒
           state.getAndSet(noRegistrationsState).asInstanceOf[Open].registrations foreach { consumer ⇒
             val newConsumers = (consumerInfo.consumers :+ consumer).sortBy(_.id)
-            val newConsumerIds = consumerInfo.consumerIds :+ consumer.id
-            Arrays.sort(newConsumerIds)
```

Comment:
Got a bit simpler now, even. I wonder if not sorting the ids now will have any consequences?